src/xcode/ENA/ENA/Source/Services/TicketValidation/AccessToken/TicketValidationAccessTokenProcessor.swift

```diff
@@ -0,0 +1,85 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import ENASecurity
+import SwiftJWT
+
+struct TicketValidationAccessTokenProcessor {
+
+	// MARK: - Init
+
+	init(jwtVerification: JWTVerifying) {
+		self.jwtVerification = jwtVerification
+	}
+
+	// MARK: - Internal
+	
+	func process(
```

Comment:
More logging would be nice. Maybe an info log when the processing starts and finished. And in between some debugging logs.